opentelemetry-android-instrumentation/build.gradle.kts

```diff
@@ -42,23 +42,22 @@ android {
     }
 }
 
-val otelVersion = "1.29.0"
-val otelAlphaVersion = "$otelVersion-alpha"
-val otelInstrumentationVersion = "1.29.0"
+val otelVersion = "1.30.0-SNAPSHOT"
+val otelAlphaVersion = otelVersion.replace("-SNAPSHOT", "-alpha-SNAPSHOT")
```

Comment:
this does not work for non snapshot versions

Reply:
ah yes, good catch! thank you.